server/novaboot-shell

```diff
@@ -124,7 +161,7 @@ run_novaboot() {
     # Split $target_config below by newlines, not by words
     IFS="
 "
-    novaboot "$nbscript" $target_config --server="$HOME/tftproot" --reset-cmd="${reset_cmd:?}" --remote-cmd="${console_cmd:?}" "$@"
+    novaboot "$nbscript" $target_config --server="$HOME/tftproot/${NB_USER:?}" --reset-cmd="${reset_cmd:?}" --remote-cmd="${console_cmd:?}" "$@"
```

Comment:
This is used for more complex configurations. I don't think we need to change it here.